j5/boards/sr/v4/power_board.py

```diff
@@ -108,7 +108,11 @@ def make_safe(self) -> None:
         self._output_group.power_off()
 
     def wait_for_start_flash(self) -> None:
-        """Wait for the start button to be pressed and flash."""
+        """
+        Wait for the start button to be pressed and flash.
+
+        The LED will remain on at the end of the flashing.
```

Comment:
```suggestion
        The LED will remain on once the start button has been pressed.
```
Would this provide more context? The original could be read as "the LED will flash until it doesn't"